bounties_api/std_bounties/views.py

```diff
@@ -103,3 +104,18 @@ def get(self, request, address=''):
             'submissions_acceptance_rate': submissions_acceptance_rate,
         }
         return JsonResponse(profile_stats)
+
+class Token(APIView):
+    def get(self, request):
+        tokens = []
+        token_to_append = {}
+        all_tokens = models.Token.objects.all()
+        for token in all_tokens:
```

Comment:
So this is going to initiate a lot of queries. Basically for each token in the set, a separate filter or lookup has to be initiated on the database.

A suggestion I'd offer is to first use some type of aggregate function. take a look at the answer with 56 upvotes here - https://stackoverflow.com/questions/3606416/django-most-efficient-way-to-count-same-field-values-in-a-query and just put tokenSymbol, tokenContract, and tokenDecimals in values and put count on tokensymbol.

Reply:
Then you can iterate through those results to do the `Token.objects.filter` lookup.

Reply:
A pro-tip here. `docker-compose exec bounties_api python manage.py shell` will let you play in a shell.

Reply:
Thank you very much for the suggestions. I have made the changes